app/src/main/res/values/strings.xml

```diff
@@ -671,6 +684,11 @@
     <string name="settings_theme_dialog_themes_3">Dark</string>
     <string name="settings_theme_dialog_themes_4">True Black</string>
 
+    <string name="settings_notification_dialog_title">Notification Action</string>
+    <string name="settings_notification_dialog_option_1">No action button</string>
+    <string name="settings_notification_dialog_option_2">STOP button</string>
+    <string name="settings_notification_dialog_option_3">RethinkDNS mode (DNS &amp; DNS+Firewall mode)</string>
```

Comment:
I don't understand from the code where `...option_` is shown in the UI as opposed to `...desc_`. The variable names are also too abstract.

Reply:
dialog_option's are for the Dialog which is shown in the settings. 
notification_desc's are to show in the Settings screen below as description.

Reply:
Okay, in that case, shouldn't the description also appear in the alert dialog? beacuse the radio buttons `Stop`, `DNS / DNS + Firewall`, `Empty` wouldn't make much sense to users without a desc in the dialog itself.

Reply:
Yes, the dialog will have the description and the description will have the selected value. 
eg., Choose which action buttons to show with the RethinkDNS notification: DNS / DNS + Firewall